applications/portfoliomanager/tests/test_risk_management.py

```
@@ -749,3 +749,318 @@ def test_create_optimal_portfolio_mixed_closed_and_maintained_positions() -> Non
 
     sorted_result = result.sort(["ticker", "side"])
     assert sorted_result.equals(result)
+
+
+def test_add_portfolio_performance_columns_rebalancing_closes_shorts_when_longs_stopped() -> (  # noqa: E501
```

Ruff (0.14.14)

[warning] Unused `noqa` directive (non-enabled: `E501`)

Remove unused `noqa` directive

(RUF100)